src/api/data/tree/importPlan.ts

```diff
@@ -189,54 +190,33 @@ const replaceRefWithId =
       Object.entries(record).map(([k, v]) => [k, v === `@${ref.refId}` ? ref.id : v])
     ) as SObjectTreeInput;
 
-export const validatePlanContents =
```

Comment:
meta: don't break the rule and then add the comment to allow you to break the rule unless you've got a really compelling reason (and preferably explain your reason!)  

here: you can export it for test only, it won't cause it to be publicly available unless you export it from index.js or some other package.json#exports file (I'm trying to guess why you'd want to do that)